protoc-gen-gripmock/generator.go

```diff
@@ -169,6 +169,10 @@ func resolveDependencies(protos []*descriptor.FileDescriptorProto) map[string]st
 				continue
 			}
 
+			// some package declarations use a semicolon, which causes a formatting error
+			// ignore for now
+			pkg = strings.Split(pkg, ";")[0]
```

Comment:
so here we should check wether it's contained a dedicated alias or not, if yes then use it as alias.